sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -775,6 +778,57 @@ def _get_sample_count(self, n_samples):
             return ceil(n_samples * self.subsample)
         return n_samples
 
+    def _get_color_dict_from_values(
+        self, value_list, categorical_or_continuous: str, cmap_name: str = "Spectral"
+    ):
+        color_dict = {}
+        if categorical_or_continuous == "categorical":
+            mapping = {
+                np.unique(value_list)[i]: f"C{i}"
+                for i in range(len(np.unique(value_list)))
+            }
+            color_dict["color_list"] = [
```

Comment:
kannst du da nicht einfach `sorted(mapping.values())` nehmen?

Reply:
Ne, weil dann kriege ich ja nen array das so lang ist wie die anzahl der kategorien. Ich brauche ja aber nen farbwert an jedem index. Damit in der plot function klar ist wie die linie eingefärbt werden soll